userscript/ui/tabSearchReports.html

```diff
@@ -44,7 +44,14 @@
               class="location-field"> -
             <input type="number" id="system_max" value="" min="1" max="400" class="location-field">
           </td>
-          <td></td>
+          <td>
+            <label for="is_moon">Mond: </label>
```

Comment:
```suggestion
            <label for="is_moon">Moon: </label>
```

for now english. If you know how to add language support, let's talk :D